LandSandBoat/server
# [SQL] Corrects War Lizard spawn positions in Davoi

**_I affirm:_**

## What does this pull request do?

 This PR aims to correct the two missing "War Lizards" spawn locations. A capture was provided in the issue for a ID reference. Currently the ID's of 17387604 & 17387605 are spawning within alleyways and this is not correct. 
Fixes https://github.com/LandSandBoat/server/issues/3758

## Steps to test these changes

 Import SQL update. See 4 War Lizards spawn in the correct retail location of J-9 in Davoi.
